config/towers.py

```diff
@@ -0,0 +1,97 @@
+"""Configuration file for towers"""
+from enum import Enum
+
+class Ammo(Enum):
+    BULLET = "Bullet"
+    LASER = "Laser"
+    ROCKET = "Rocket"
+    SNOWFLAKE = "Snowflake"
+
+class Ability(Enum):
+    ATTACK_ALL = "Can attack all enemy types"
+    SPLASH_DAMAGE = "Splash damage"
+    FREEZE_ENEMIES = "Freeze enemies"
+    DESTROYS_CAMOUFLAGE = "Destroys camouflage, therefore they act like normal enemies"
```

Comment:
Ten popis abilit davej treba do komentare, ne primo do toho Enumu, to je strasne matouci, kdyz clovek potrebuje tu samou abilitu hledat podle stringu (nikdo by necekal, ze pod ATTACK_ALL bude ulozena veta "Can attack all enemy types"). Navic si predstav, ze to musis pouzit v kodu. To budes psat **if object.ability.value == "Destroys camouflage, therefore they act like normal enemies"**?

Idealne, at ten string je to jmeno, takze treba ATTACK_ALL = "attack_all" (nemit tam velka pismena je taky lepsi, casty zdroj chyb, ale to uz ti nutit nebudu).